cmd/eksctl-anywhere/cmd/getpackage.go

```diff
@@ -0,0 +1,67 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+
+	"github.com/aws/eks-anywhere/pkg/executables"
+	"github.com/aws/eks-anywhere/pkg/features"
+)
+
+type getPackageOptions struct {
+	output string
+}
+
+var gpo = &getPackageOptions{}
+
+func init() {
+	getCmd.AddCommand(getPackageCommand)
+	getPackageCommand.Flags().StringVarP(&gpo.output, "output", "o", "", "Specifies the output format (valid option: json, yaml)")
+}
+
+var getPackageCommand = &cobra.Command{
+	Use:     "package(s) [flags]",
+	Aliases: []string{"package", "packages"},
+	Short:   "Get package(s)",
+	Long:    "This command is used to display the installed packages",
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if !features.IsActive(features.CuratedPackagesSupport()) {
+			return fmt.Errorf("curated Packages not supported in this release")
+		}
+		cmd.Flags().VisitAll(func(flag *pflag.Flag) {
+			if err := viper.BindPFlag(flag.Name, flag); err != nil {
+				log.Fatalf("Error initializing flags: %v", err)
+			}
+		})
+		return nil
+	},
+	SilenceUsage: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		packageInstanceName := ""
+		if len(args) > 0 {
+			packageInstanceName = args[0]
```

Comment:
Let's have a separate validate function that is a bit more robust and can fail if more than one arg is specified

Reply:
Is there any reason we want to fail in case more than the necessary args are specified? The validation for our use case is to a minimal and if there is a necessity for more validation, it is a good idea to move it to a function but I don't see that here in this case.